<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="8,1,1">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QDial" name="pDial">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3,2,3">
       <item>
        <widget class="QLabel" name="pLabelLed">
         <property name="text">
          <string>LED</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pLabelLedVal">
         <property name="text">
          <string>LED Val</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pLabelKey">
         <property name="text">
          <string>KEY</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pLabelKeyVal">
         <property name="text">
          <string>Key Val</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QProgressBar" name="pProgressBar">
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="value">
        <number>25</number>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
